### ISSUE TITLE
Fucking cardinal directions

### ISSUE THREAD

**Issue by Riekland** [d3athrow/vgstation13#13575]
There's no fucking north in space
As far as I know, the meteor alerts, solars, and possibly power use NSEW instead of fore, aft, starboard, port
The fact of the matter is that it encourages the kind of person who takes an engineering role to not learn which way is which on a bloody ship, which leads to !!FUN!! when trying to direct them to breaches without sounding like a retard
Please to be fixing posthaste

**Comment by despotate** [comment]
My fix for this would be to remove the "fore, aft, starboard" bullshit because the massive amount of confusion they cause time and time again isn't worth the immulsions.

Plus, doesn't the space station orbit a gas giant? If it were to be tidally locked to it, then we could definitely have NSEW directions.

**Comment by Aurx** [comment]
>There's no fucking north in space

There's north anywhere you've got a magnetic field and a sensitive enough compass.

**Comment by ihadtoregisterforthis** [comment]
>way is which on a bloody ship

This isn't a ship.

Cardinal directions are a clear way of saying top, bottom, left and right of the screen, while staying IC

**Comment by SonixApache** [comment]
Why would you use naval directions on a station anyway? NWSE works just fine. Maybe we should just start renaming sections to the NWSE system.

I think I saw a round on bay where an AI described where something was with "It is Aft of the aft-bow x", shit sounds retarded.

**Comment by ComicIronic** [comment]
Because ship directions are station-relative, and it makes sense to use them to refer to directions in the thing you're moving in. Since the station is in planetary orbit, NSEW would be directions on the surface not necessarily aligned with the station's directions. If the sun's position is to be believed, the station is actually flat on the system's orbital plane.

We should settle on one system, and for the sake of sci-fi, I think it should be the naval one.

**Comment by Kitfox88** [comment]
i like nsew because im a lazy sack of shit

**Comment by Ephx1** [comment]
> A player is facing downward, character sprite faced towards the screen
> Tell them left
> Immediately turns right at the screen's left

Every time, this is why we have NESW

**Comment by 9600bauds** [comment]
![maint naming guide](https://cloud.githubusercontent.com/assets/6526157/22179909/3026e372-e03f-11e6-822c-9d885e89c991.png)

**Comment by PJB3005** [comment]
I'm with Comic here.

**Comment by Shadowmech88** [comment]
Given that the station has no propulsion, and so can't really have a "front" and "back" and all that, using nautical terminology doesn't exactly make *total* sense.
That said, it makes more sense than using NSEW, and it sounds better.
I'm for standardizing to nautical terminology as well.

**Comment by Exxion** [comment]
The station is probably flat on the plane of the system (as Comic said) AND tidally-locked to the planet (so the gas miners work)
Which would make east and west well-defined, but not north or south
Therefore, I propose we use east, west, up, and down

(The point is that it doesn't matter)

**Comment by Exxion** [comment]
Also, BYOND uses NSEW

**Comment by ComicIronic** [comment]
A lot of stuff happens in code which isn't reflected in the game world. Every radio doesn't have a mini leprechaun in it in the lore.

**Comment by Exxion** [comment]
Let's fix that

**Comment by ComicIronic** [comment]
In the 41st millennium, capitalists have harnessed old Irish magic folk to power their faster-than-light communication equipment.

**Comment by uazuaz** [comment]
i like naval terms, but remind about this:

![1](https://cloud.githubusercontent.com/assets/6170726/22284507/eb950102-e2e7-11e6-8b3f-b90ce8d8ab5e.jpg)

**Comment by Exxion** [comment]
If you want to know my honest, non-meme opinion, I kind of prefer NSEW because this is a "medium" (low by any standard except SS13) RP server and NSEW is easier and more natural for most people.

**Comment by Intigracy** [comment]
Or we can force people to learn something ;^)

**Comment by gbasood** [comment]
haaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

**Comment by Intigracy** [comment]
I bothered to learn naval directions because of ss13.

You should too!

**Comment by Shadowmech88** [comment]
We should frame that picture of the boat with the directions on it and hang it on a wall in Arrivals.

**Comment by uazuaz** [comment]
what if holomaps show the naval terms on the 4 sides of the map?

**Comment by uazuaz** [comment]
reminder.

![nsew](https://cloud.githubusercontent.com/assets/6170726/23178951/8cf6bda8-f86c-11e6-94f0-9e24893e542f.jpg)

**Comment by PJB3005** [comment]
We should fix those question marks. We have the technology!

**Comment by uazuaz** [comment]
it should also be fixed the firelock checking for all 4 directions instead of just 2 directions.

**Comment by PJB3005** [comment]
It only checks two directions if the others are blocked, which is common with airlocks.

Should be working as intended.

**Comment by Probe1** [comment]
It's inherited code from when airlocks were a tile large.

**Comment by PJB3005** [comment]
Firelocks*

**Comment by gbasood** [comment]
rimwards hubwards widdershins etc tbh

**Comment by Exxion** [comment]
You know, thinking about it more, nothing actually makes sense about the station's orbit
1. It rotates such that its axis of rotation is perpendicular to the line between it and the system's star
2. It orbits a gas giant but is never eclipsed by it
3. It is presumably tidally-locked to said gas giant so station-mounted mining equipment works, though I guess if they're drone miners or something this isn't strictly necessary

The only orbit I can imagine that satisfies all these criteria is a polar orbit around the gas giant in the plane perpendicular to the line between the gas giant and the star (which would change gradually throughout the year) with the station itself rotating around the radial of the orbit
Which is possible (at least with a constant fuel expenditure), but nonsensical
If you're going to go through the trouble of maintaining that orbit (which, to be fair, is a useful orbit for solar power), why rotate about the radial? If you kept constant heading towards the star instead, you could mount one big stationary set of panels rather than a bunch of small articulated ones.

In that orbit, the cardinal directions would (usually) all be well-defined, but would constantly be changing in relation to the station

Alternatively, the station could be in one of the gas giant's Lagrange points rather than in orbit, and be flat and rotating on the system's orbital plane, but that seems like a really long distance for mining purposes. That said, it would explain the asteroid near the station.

TL;DR: NSEW doesn't actually make sense in any possible orbit the station could have.
But naval directions are equally meaningless on a craft without engines.
NSEW is easier for people to remember, and this is a low RP server, so I'd still go with that

**Comment by SonixApache** [comment]
just pretend the station's gravity comes from magnets
and their north poles are all pointed towards the fore

**Comment by Shadowmech88** [comment]
>It orbits a gas giant but is never eclipsed by it

What if the gas giant orbits its sun at exactly the speed such that the station is always between the gas giant and the sun?

**Comment by Exxion** [comment]
That would be the L1 Lagrange point

**Comment by Shadowmech88** [comment]
What if the gas giant is tilted so that its poles are on the orbital plane, but rotating on an axis perpendicular to the poles, and the station is perpetually above it in an orbit around the sun, also rotating on that axis?

So the directions on the station relative to the gas giant are always the same *and* each side of the station is periodically bathed in sunlight as it rotates on its axis.

**Comment by ComicIronic** [comment]
I want a diagram of what you just described

**Comment by PJB3005** [comment]
I'm telling you this is turning into literal rocket science.

Just do a server vote I'd say.

**Comment by Shadowmech88** [comment]
Something like this.
![diagram](https://cloud.githubusercontent.com/assets/3991927/24684552/80fd3150-196c-11e7-878d-a9d1dd92fa96.png)

**Comment by Aurx** [comment]
How the literal fuck are you suggesting the magnetic poles be orthogonal to the axis of rotation?  
I mean, how would that even work?

**Comment by Shadowmech88** [comment]
I don't know a whole lot about orbital mechanics.
Does the orientation of the magnetic poles influence the axis of rotation or vice versa?

**Comment by ComicIronic** [comment]
You can't have an orbit like that. The plane of the orbit always cuts through the centre of the parent body.

**Comment by Aurx** [comment]
> Does the orientation of the magnetic poles influence the axis of rotation or vice versa?

The rotation of the planet is what generates the magnetic field and thus defines the poles.

**Comment by Exxion** [comment]
The poles are 100% possible. Source: Uranus
When they say Uranus is sideways, it's truer than you probably thought
Its magnetic poles are closer to its equator than to its axis of rotation

**Comment by Aurx** [comment]
>off from the gravitational center by a third of the radius

What the literal shit

**Comment by Exxion** [comment]
Yeah that part is very impossible

**Comment by Aurx** [comment]
No, I mean Uranus's magnetic center is different from it's gravitational center.  
By a fucking third of its radius.  

It's fucking inexplicable to me, so I'm going to swing by the astrophysics department on Friday and start bothering random grad students until I get an answer.

**Comment by SonixApache** [comment]
post here when you do

**Comment by PJB3005** [comment]
This is getting into the range of when Polaris asked NASA about something.

**Comment by Commandersand** [comment]
Fuck it dude, someone go email nasa

**Comment by ihadtoregisterforthis** [comment]
NASA would say boat-like directions, but the ISS isn't a topdown 2D full of casuals

**Comment by Exxion** [comment]
NASA would not say naval directions for a craft without engines

**Comment by Shadowmech88** [comment]
How do they reference directions onboard the ISS?
The ISS is a space station.

This may actually warrant asking NASA.

**Comment by ihadtoregisterforthis** [comment]
https://www.nasa.gov/mission_pages/station/multimedia/iss_labs_guide.html

**Comment by Exxion** [comment]
Well fuck me then

**Comment by ComicIronic** [comment]
Official burger space directions

**Comment by Exxion** [comment]
Technically the alignment of the directions is arbitrary, but I guess they just... picked one

**Comment by Aurx** [comment]
I went by the astrophysics building today.  

Freshmen parted before me like the red sea, and flocks of grad students ducked into the nearest labs with panic in their eyes.  Soon, however, I found prey.  A lone professor, his back turned as he closed and locked his office door.  He was denying himself escape.  

I pounced, like a bolt of lightning from a clear sky.  "Why," I began as the professor realized he had met his end, "is it that Uranus has magnetic poles over fourty-five degrees away from its rotational poles?"  He started to stammer out an excuse, that his office hours were over, but each word left his tongue like it was the only thing standing between himself and hell.  I paid him no heed, and continued: "And moreover, why is the center of Uranus's magnetic field a full third of the planet's radius away from its center of mass?"  He froze, a thin sheen of sweat forming on his brow.  His heart heavy, like a man announcing his own death, he answered.  

"I'd have to look that up."

**Comment by gbasood** [comment]
🤔

**Comment by PJB3005** [comment]
WELL if NASA uses something close to naval directions for the ISS, I think we've decided.

**Comment by Exxion** [comment]
Nah fuck that
This is a game
People are going to still use NSEW in-game, so why go against it?

**Comment by PJB3005** [comment]
Of course they're not gonna use naval directions if half the directions are NSEW.

People in America aren't gonna switch to using kilometers either until you change all the road signs.

**Comment by SonixApache** [comment]
What NASA uses is nice and all but what do ESA and Roscosmos use?

**Comment by ihadtoregisterforthis** [comment]
The ISS page says the Zvezda module is the back, and Wikipedia states that
>The two main engines on Zvezda can be used to raise the station's altitude.

So they might be using engines to define the coordinates

**Comment by Shadowmech88** [comment]
I imagine the ISS has engines on all sides though, for course correction and debris avoidance.

**Comment by Exxion** [comment]
People in-game are never going to stop using NSEW, regardless of what the signs say in-game
Because that is the language used in basically every other topdown game ever
